inputfiles/overridingTypes.jsonc

```diff
@@ -3233,12 +3243,14 @@
                         "readable": {
                             "name": "readable",
                             "readonly": true,
-                            "overrideType": "ReadableStream<Uint8Array>"
+                            "type": "ReadableStream",
+                            "subtype": { "type": "Uint8Array" }
                         },
                         "writable": {
                             "name": "writable",
                             "readonly": true,
-                            "overrideType": "WritableStream<string>"
+                            "type": "WritableStream",
```

Comment:
The output doesn't change that I could tell, so I guess this is an update to use the new subtype property for consistency?

Reply:
Refraining from overrideType is a good change 👍🏻 (use of overrideType disables type dependency tracking)

Reply:
(Though I would have preferred if this kind of change is file as a separate PR, as mixing it makes harder to understand it)